### DISCUSSION TITLE
1 fc cable (48 pairs) is connected to 3 patch-panels (12pairs)

### DISCUSSION THREAD

**Discussion by frankie64** [netbox-community/netbox#15750] [General]
Hi,

How can I document in netbox that a fiber cable with 48 pairs is connected to 3 fc patch-panels with 12 pairs each?

In addition, these 3 panels are installed in a plug-in frame. For this netbox knows bays and modules. But I can't define a comprehensive rear port with 48 distributed ports there?

Complicated and I still have no idea how to display this.

Anybody as an idea?

thanks for help

**Comment by a084ed22** [comment]
There's no representation that will give you exactly that.
Closest thing you can obtain is to make a single patch panel spanning 3 units, if you want to preserve the information about having a single 48-pair fiber as a rear port.

Alternatively, if you prefer to keep the information about having 3 separate patch panels, you'll have to pretend that you have different fibers as rear ports for each patch panel.

**Reply by frankie64** [reply]
thx for your ideas. Now we used your first idea. we have configured on the bay a rearport with 48 positions and 48 frontport. the modules got no ports. but we labeled them like 'Modul1 (1..12)' ....

**Comment by cpmills1975** [comment]
It would be awesome to be able to model multi-ended cables from and to multiple rear ports in this way.

Something like this: https://ecatalog.corning.com/optical-communications/SEA/en/Fiber-Optic-Cable-Assemblies/Indoor-Cable-Assemblies/Multifiber-Indoor-Cable-Assemblies/EDGE8®-MTP®-Trunk-Cable/p/GE5E548QLZDDU030M

The problem is that while front ports map on to a position on the rear port, the rear port carries all of those positions. Positions would need to move to the cable model somehow and a rear port would map on to just some of the positions of the cable.

**Reply by frankie64** [reply]
oh, thats bad news. I hoped, if netbox goes  in 'pair no 3' in this cable he comes out on 'pair no 3' on the other side. Thats the normal way, I believe. Than the cable path doesn't show the correct way?

**Reply by cpmills1975** [reply]
The front port records its position on the rear port and providing that rear port is connected via a cable to one other rear port, that position is carried through just fine - think 12 LC ports being bundled on to a 12 strand MPO and then coming back out as another 12 LC ports. No problem.

The problem occurs when multiple MPO 12s are linked to multiple MPO 12s using the type of cable shown. There is no way to track which end of the multi-end cable goes where.

The only way I've been able to record this is to create multiple cables with the same label on them.

**Reply by frankie64** [reply]
ok, thanks for your detailed explanations. Than I have no problems. ~
